cl/lib/recap_utils.py

```diff
@@ -120,19 +120,33 @@ def needs_ocr(content):
         "Appellate",
         "Appeal",
         "Case",
+        "Desc",
+        "Main Document",
         "Page",
+        "Received:",
         "USCA",
     )
     pagination_re = re.compile(r"Page\s+\d+\s+of\s+\d+")
+    doc_filed_re = re.compile(r"Doc\s+\d+\s+Filed")
+    case_num_re = re.compile(r"^\d+:\d+-CV-\d+$")
```

Comment:
I think you added this one for the example related to `1:16-CV-107 Received: 06/03/2016`, right?

If that's the case, this type of header would be handled by the "Received" match suggested above.

Also, since this is matched using `case_num_re.match(line)`, it doesn’t match the real example, as the pattern only matches the docket number when it appears alone on a line.

Unless there are examples where matching the docket number is actually helpful (in which case we should also consider appellate and bankruptcy formats), I think we could remove this one.